data/fields/railway/signal/railway:signal:position

```diff
@@ -0,0 +1,6 @@
++ {
```

Comment:
As described here: https://github.com/ideditor/schema-builder#fields, the filename should be: position.json

Additionally, I think this field should be added to the corresponding preset as a strong field. (presets/railway/signal.json)